src/App.java

```diff
@@ -2,30 +2,209 @@
+        System.out.println();
+    }
+
+    public void doMadlib1(){
+        Scanner getMLInfo = new Scanner(System.in);
+
+        System.out.print("Enter a name: ");
+        String nameML = getMLInfo.nextLine();
+
+        System.out.print("Enter a place: ");
+        String placeML = getMLInfo.nextLine();
+
+        System.out.print("Enter an adjective: ");
+        String adjective1ML = getMLInfo.nextLine();
+
+        System.out.print("Enter an adverb: ");
+        String adverbML = getMLInfo.nextLine();
+
+        System.out.print("Enter an animal: ");
+        String animalML = getMLInfo.nextLine();
+
+        System.out.print("Enter a clothing item: ");
+        String clothingItemML = getMLInfo.nextLine();
+
+        System.out.print("Enter an adjective: ");
+        String adjective2ML = getMLInfo.nextLine();
+
+        System.out.print("Enter a plural noun: ");
+        String pluralNounML = getMLInfo.nextLine();
+
+        System.out.print("Enter a type of music: ");
+        String musicML = getMLInfo.nextLine();
+
+        System.out.print("Enter an adjective: ");
+        String adjective3ML = getMLInfo.nextLine();
+
+        System.out.print("Enter a verb: ");
+        String verb1ML = getMLInfo.nextLine();
+
+        System.out.print("Enter a verb: ");
+        String verb2ML = getMLInfo.nextLine();
+
+        System.out.print("Enter an adjective: ");
+        String adjective4ML = getMLInfo.nextLine();
+
+        System.out.print("Enter a number: ");
+        String numberML = getMLInfo.nextLine();
+
+        System.out.println();
+        System.out.println("Here is your very own story!");
+        System.out.println();
+
+        System.out.println("One day, my best friend " + nameML);
+        System.out.println("and I decided to go to the " + placeML + ".");
+        System.out.println("It was a/an " + adjective1ML);
+        System.out.println("day, and the sun was shining " + adverbML + ".");
+        System.out.println("As soon as we walked through the gates, we saw a/an " + animalML);
+        System.out.println("wearing a/an " + clothingItemML + "!");
+        System.out.println("We couldn't believe our eyes.");
+        System.out.println();
+        System.out.println("Next, we headed over to the " + adjective2ML);
+        System.out.println("exhibit, where we saw a group of " + pluralNounML);
+        System.out.println("dancing to " + musicML + ".");
+        System.out.println("It was so " + adjective3ML);
+        System.out.println("that we started to " + verb1ML);
+        System.out.println("along with them.");
+        System.out.println();
+        System.out.println("As the day came to an end, we decided to " + verb2ML);
+        System.out.println("the bus all the way home. It was truly a/an " + adjective4ML);
+        System.out.println("day at the " + placeML + ",");
+        System.out.println("and we promised to come back again in " + numberML + " weeks!");
+    }
+
+    public void doMadlib2(){
```

Comment:
I like that you broke this into separate methhods, it definitely aids readability